frontend/src/components/experimentRuns/ExperimentRunsListContent.js

```diff
@@ -80,14 +73,24 @@ const getTableColumns = (containersByID, runTypes, instruments) => [
     sorter: true,
     width: 180,
   },
-
+  {
+    title: "Launch",
+    dataIndex: "run_processing_launch_date",
+    sorter: true,
+    render: (_, experimentRun) => (
+      <div style={{minWidth: "12rem"}}>
+        <ExperimentRunLaunchCard experimentRun={experimentRun} experimentRunLaunch={launchesById[experimentRun.id]} launchExperimentRun={launchExperimentRun}/>
```

Comment:
isn't the parameter `launchExperimentRun` superfluous? I might be wrong though.

Reply:
Yup. Originally I was passing the function as a prop, but now I just import the function in the card component.